frontend/src/app/task-area/task-area.component.ts

```diff
@@ -1,15 +1,43 @@
 import { Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
 import { TaskLoaderService, TaskInterface } from '../task-loader.service';
+
 @Component({
   selector: 'app-task-area',
   templateUrl: './task-area.component.html',
   styleUrls: ['./task-area.component.scss'],
 })
 export class TaskAreaComponent implements OnInit {
-  constructor(private readonly taskLoaderService: TaskLoaderService) { }
+  constructor(private readonly taskLoaderService: TaskLoaderService) {}
   displayedTasks: TaskInterface[] = [];
+  currenTab: string = 'All';
+
+  taskText = new FormControl('', [
+    Validators.required,
+    Validators.minLength(4),
+  ]);
 
   ngOnInit(): void {
     this.displayedTasks = this.taskLoaderService.getItems();
   }
+
+  checkTask(taskInformation:TaskInterface) {
```

Comment:
It's necessary to write what does function return
func ()**: void** { }

and in all other functions